src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,171 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition.

Reply:
Ty, but it's an issue here 
![Scrn](https://github.com/user-attachments/assets/dbfa3ce6-cc32-4ad0-9f42-33079316f342)